nebari/template/stages/02-infrastructure/aws/main.tf

```diff
@@ -71,16 +79,13 @@ module "kubernetes" {
   region             = var.region
   kubernetes_version = var.kubernetes_version
 
-  cluster_subnets         = module.network.subnet_ids
-  cluster_security_groups = [module.network.security_group_id]
+  cluster_subnets         = local.subnet_ids
+  cluster_security_groups = [local.security_group_id]
 
   node_group_additional_policies = [
     "arn:aws:iam::aws:policy/AmazonEC2ContainerRegistryReadOnly"
   ]
 
   node_groups = var.node_groups
 
-  depends_on = [
-    module.network
-  ]
```

Comment:
This is unnecessary since terraform can deduce the dependency from the (`module.network` (previously, now it's `local.subnet_ids`) variable)